cmd/node/node.go

```diff
@@ -457,7 +458,7 @@ func (app *App) initServices(ctx context.Context,
 		}),
 		txs.WithLogger(app.addLogger(ConStateLogger, lg)))
 
-	genesisAccts := app.Config.Genesis.ToAccounts()
+	genesisAccts := app.Config.Genesis.Accounts.ToAccounts()
```

Comment:
Thinking if `app.Config.Genesis.Accounts.ToList()` makes more sense here as you convert accounts map to a list here 🤔